frontend/src/metabase/collections/components/ItemList.jsx

```diff
@@ -77,18 +62,34 @@ export default function ItemList({
       )}
       {items.length > 0 && (
         <PinDropTarget pinIndex={null} margin={8}>
-          <Box
-            style={{
-              position: "relative",
-              height: ROW_HEIGHT * items.length,
-            }}
-          >
-            <VirtualizedList
-              items={items}
-              rowHeight={ROW_HEIGHT}
-              renderItem={renderListItem}
-              scrollElement={scrollElement}
-            />
```

Comment:
As we don't need to use `VirtualizedList` here anymore, we should also delete `scrollElement` from other places. I added it at #15715 to fix list virtualization, but now we can remove it completely. It's passed from the `CollectionLanding` down here through `CollectionContent`